torch/distributed/_tensor/op_schema.py

```diff
@@ -121,26 +126,24 @@ def output_ndim(self):
 class TupleStrategy(StrategyType):
     """
     TupleStrategy represents the output strategy of this op is a tuple
-    of strategy, i.e. If the output of this op is a tuple of tensors, we should
-    return a TupleStrategy that contains a tuple of OpStrategy.
+    of strategy, i.e. If the output of this op is a tuple of tensors or list of tensors
+    with possibly different placement strategies, we should return a TupleStrategy that
+    contains a tuple of OpStrategy.
 
-    NOTE: if the output of the op is a List[Tensor], it's likely we should return
-    OpStrategy directly in all cases.
+    NOTE: if the output of the op is a List[Tensor] and they share the same placement
+    strategy, then we should return a single OpStrategy instead of a TupleStrategy
     """
 
-    def __init__(self, childs: Tuple[StrategyType, ...]) -> None:
+    def __init__(self, childs: Sequence[StrategyType]) -> None:
         super().__init__()
-        self.childs: Tuple[StrategyType, ...] = childs
+        self.childs: Sequence[StrategyType] = childs
 
     def __str__(self) -> str:
-        tuple_strategies_str = "TupleStrategy: "
-        child_strategies_str = "\n".join(
-            [
-                f" tuple idx: {idx}, strategy: {str(strat)}"
-                for idx, strat in enumerate(self.childs)
-            ]
+        tuple_strategies_str = "TupleStrategy("
+        child_strategies_str = ", ".join(
+            [f"{str(strat)}" for idx, strat in enumerate(self.childs)]
         )
-        return f"{tuple_strategies_str}\n{child_strategies_str}"
+        return f"{tuple_strategies_str}{child_strategies_str})"
```

Comment:
maybe change to
```
child_strategies_str = ", ".join(...)
return f"TupleStrategy({child_strategies_str})"
```
for a better readability since we only use `tuple_strategies_str` once? Just my personal opinion when reading the code.

Reply:
Yeah, sure, will submit follow up PR to avoid trigger CI again